Azure/README.md

```diff
@@ -0,0 +1,37 @@
+# Clear Containers on Azure-VM
+
+This project allows to launch an Azure VM and install all
+stuff in order to run Clear Containers.
```

Comment:
I'd replace `all stuff in order to run Clear Containers.` with something like:

'install all Clear Container components and dependencies`